app/main/api/handlers/newMnemonics.ts

```diff
@@ -0,0 +1,20 @@
+import { asObject } from "app/common/runtimeTypes"
+import { SubSystems } from "app/main/system"
+import { generate } from "app/main/crypto/mnemonic"
+import { NewMnemonicsResponse } from "app/common/runtimeTypes/storage/wallets"
+
+/**
+ * Handler function for "newMnemonics" method.
+ * It retrieves and returns a list of mnemonics words.
+ * @param system
+ * @param params
+ */
+export default async function newMnemonics(system: SubSystems, params: any) {
+  // Get mnemonics from crypto library
+  const mnemonicsResponse = { mnemonics: generate() }
```

Comment:
We could type this as `NewMnemonicsResponse` so that we can guarantee its type safety at compile time.